style_editor.html

```diff
@@ -9525,7 +9525,7 @@
 /* Variant Slider functions */
 
 function updateVariantValue(newValue) {
-    if (newValue < 0) {
```

Comment:
Under what circumstances would this make a difference?

Reply:
I have a safeguardInputs() function coming that will prevent NaNs.
It autofills in a zero in the case where the user deletes the value completely.
WavLen will fill in a 1 as a minimum.
This one character fix for now allows the autofilled zero to be taken into account.